transactions/lockedTokens/admin/unlock_tokens_for_multiple_accounts.cdc

```diff
@@ -56,7 +56,11 @@ transaction(unlockInfo: {Address: UFix64}) {
 
         admin.storage.save<{Address: UFix64}>(badAccounts, to: /storage/unlockingBadAccounts)
 
-        let unlockingBadAccountCap = admin.capabilities.storage.issue<&{Address: UFix64}>(/storage/unlockingBadAccounts)
-        admin.capabilities.publish(unlockingBadAccountCap, at: /public/unlockingBadAccounts)
+        let unlockingBadAccountCap = admin.capabilities.get<&{Address: UFix64}>(/public/unlockingBadAccounts)
+        if !unlockingBadAccountCap.check() {
+            let unlockingBadAccountCap = admin.capabilities.storage.issue<&{Address: UFix64}>(/storage/unlockingBadAccounts)
```

Comment:
i guess the extra `let` is just shadowing this var, which isn't a problem since it's only used here in this if.

Reply:
yeah, shouldn't be a problem to shadow it